src/components/header/Header.js

```diff
@@ -0,0 +1,29 @@
+import Navbar from 'components/navbar'
+import { getUser } from 'contexts'
+import React from 'react'
+import './Header.scss'
+import AccountCircleIcon from '@mui/icons-material/AccountCircle'
+
+const Header = () => {
+  const user = getUser()
+  return (
+    <div>
+      <div className='menu-bar'>
+        {!user ? (
+          <a href='/login'>Login</a>
+        ) : (
+          <>
+            <AccountCircleIcon className='icon' fontSize='large' />
+            {user} | <a href='/logout'> Logout</a>
+          </>
+        )}
+      </div>
+      <header>
+        <h1 className='brand'>Thao Ho Blog</h1>
+      </header>
+      <Navbar user={user} />
+    </div>
```

Comment:
The header should contain the brand and the user info. So this component should be wrapped by the header tag and the nav should move out, to the App component. https://www.w3schools.com/html/html5_semantic_elements.asp

I know you move the navbar here to avoid getting the user many times. But I think moving it out is better for structuring rather than convenience.

Reply:
thanks, fixed